packages/core/http/core-http-router-server-internal/src/router.ts

```diff
@@ -200,25 +200,30 @@ export class Router<Context extends RequestHandlerContextBase = RequestHandlerCo
     try {
       kibanaRequest = CoreKibanaRequest.from(request, routeSchemas);
     } catch (e) {
+      this.log.error(`400 Bad Request - ${request.path}${queryString(request.query)}`);
+      this.log.error(e);
```

Comment:
Minor thing: is it possible to combine these into one log?

With our logger interface we have a `meta` argument that can be passed in:       `this.log.error(e, { url: { query, path } });`. Does that result in anything useful?